pkg/apis/openebs.io/v1alpha1/cstor_volume_replica.go

```diff
@@ -72,9 +72,15 @@ const (
 	CVRStatusOffline CStorVolumeReplicaPhase = "Offline"
 	// CVRStatusDegraded means that the rebuilding has not yet started.
 	CVRStatusDegraded CStorVolumeReplicaPhase = "Degraded"
-	// CVRStatusRebuilding means that the volume is in re-building phase.
+	// CVRStatusNonQuorumDegraded means that replica(i.e recreated/scaledup
```

Comment:
It will be good to have more verbose/elaborate description. It should convey everything. This is a good time to revist the comments against different Phases & provide elaborate details.

Reply:
Updated the other Phases comments also